src/models/accounting_period.py

```diff
@@ -36,13 +36,13 @@ def as_query_params(self) -> list[QueryParameter]:
 
     def dashboard_url(self) -> str:
         """Constructs Solver Accounting Dashboard URL for Period"""
-        base = "https://dune.com/gnosis.protocol/"
-        slug = "solver-rewards-accounting-v2"
+        base = "https://dune.com/bh2smith/"
+        slug = "cow-solver-rewards"
         query = f"?StartTime={self.start}&EndTime={self.end}"
         return base + urllib.parse.quote_plus(slug + query, safe="=&?")
 
     def unusual_slippage_url(self) -> str:
         """Returns a link to unusual slippage query for period"""
-        base = "https://dune.com/queries/645559"
+        base = "https://dune.com/queries/1688044"
```

Comment:
The unusual slippage query is almost identical to the slippage query except that it has some constraints on the final select statement. In Dune V1 version of this project we had control over this and could "append" different query strings to an existing query (to avoid redundancy). Unfortunately now, we no longer have such control.

While technically we could continue using the legacy `duneapi` for its "dashboard manager" purposes, it does not jive well with the rest of the project code for the time being. The most straightforward thing we can do at this point is just live with the fact that the unusual slippage query have a different ID than the `period_slippage` query (contained in this repository). It is easy enough to check (since the URL is right here).